tests/try_into.rs

```diff
@@ -0,0 +1,89 @@
+#![allow(dead_code)]
+
+// TryInto is still an unstable trait. In the meanwhile, use a trick
+// to define the trait (as well as TryFrom) without relying on
+// #![feature(try_from)].
+#![no_std]
+extern crate std as ruststd;
+mod std {
+    pub mod convert {
```

Comment:
I think it's nicer to just not run these tests on stable for now by using this solution to that: https://stackoverflow.com/a/37680212/2570866